R/01.R

```diff
@@ -139,6 +139,68 @@
   return(valid)
 }
 
+
+# function .rc() prepends "#'" to a string or character vector
+.rc <- function(strVec){paste("#'", strVec)}
+
+# function .autoDoc() automates the creation of a basic roxygen template for the package and each object in objectsToKeep
+# arguments are pname and ds2kp, normally defined in datasets.R
+# pname is name of package, ds2kp is list of objects to save in data package
+.autoDoc <- function(pname, ds2kp, env){
```

Comment:
dataEnv and the .autodoc() function environment are both children of .GlobalEnv. Inside the function, topenv() is the .autoDoc() function environment and dataEnv is invisible. My best solution (workaround?) is to pass dataEnv as an argument into the function. From .GlobalEnv, topenv() correctly points to dataEnv when calling the function.